src/app/about/page.js

```diff
@@ -1,5 +1,27 @@
+"use client";
+import Logo from "../../../public/assets/logo.svg";
+import Image from "next/image";
+import { sections } from "@/data/aboutSections/Sections.js";
+
 const About = () => {
-  return <>About</>;
+  return (
+    <div>
```

Comment:
![image](https://github.com/acm-ucr/isa-website/assets/91030482/e2c59984-89bd-4ab0-973d-415124265f80)
add a little more margining to the bottom of page